templates/components/notepad/notes.html.twig

```diff
@@ -0,0 +1,70 @@
+{#
+ # ---------------------------------------------------------------------
+ # GLPI - Gestionnaire Libre de Parc Informatique
+ # Copyright (C) 2015-2022 Teclib' and contributors.
+ #
+ # http://glpi-project.org
+ #
+ # based on GLPI - Gestionnaire Libre de Parc Informatique
+ # Copyright (C) 2003-2014 by the INDEPNET Development Team.
+ #
+ # ---------------------------------------------------------------------
+ #
+ # LICENSE
+ #
+ # This file is part of GLPI.
+ #
+ # GLPI is free software; you can redistribute it and/or modify
+ # it under the terms of the GNU General Public License as published by
+ # the Free Software Foundation; either version 2 of the License, or
+ # (at your option) any later version.
+ #
+ # GLPI is distributed in the hope that it will be useful,
+ # but WITHOUT ANY WARRANTY; without even the implied warranty of
+ # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ # GNU General Public License for more details.
+ #
+ # You should have received a copy of the GNU General Public License
+ # along with GLPI. If not, see <http://www.gnu.org/licenses/>.
+ # ---------------------------------------------------------------------
+ #}
+
+{% set rand = random() %}
+{% set entity = get_item('Entity', item.fields['entities_id']) %}
+
+{% set notes = notepad.getAllForItem(entity) %}
+
+{% for note in notes %}
+    {% set id = "note#{ note['id'] }#{ rand }" %}
+    <div class='boxnote' id='view{{ id }}'>
+        <div class='boxnoteleft'>
+            {% set thumbnail_url = user.getThumbnailURLForPicture(note['picture']) %}
+            {% set user = get_item('User', note['users_id_lastupdater']) %}
+            {% if thumbnail_url is not empty %}
+                {% set style = "background-image: url(\"#{ thumbnail_url }\")" %}
+            {% else %}
+                {% set style = "background-color: #{ user.getUserInitialsBgColor() }" %}
+            {% endif %}
+            <a href="{{ user.getLinkURL() }}">
+                {% set user_name %}
+                    {% do call('User::formatUserName', [
+                        user.fields['id'],
+                        user.fields['name'],
+                        user.fields['realname'],
+                        user.fields['firstname']
+                    ]) %}
+                {% endset %}
+                <span class="avatar avatar-sm rounded" style="{{ style }}" title="{{ user_name }}">
+                    {% if thumbnail_url is empty %}
+                        {{ user.getUserInitials() }}
+                    {% endif %}
+                </span>
+            </a>
```

Comment:
I guess it could be replaced by
```{{ include('components/user/picture.html.twig', {'users_id': note['users_id_lastupdater']}, with_context = false) }}```